recipes/django-unicorn/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "django_unicorn" %}
+{% set version = "0.50.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 181aeb6ea51b26f37d3e6be2de76b1831f24c02611a9f55cbc59ebebda4e84b4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.7, <4
+    - pip
+    - poetry-core
+  run:
+    - python >=3.7, <4
+    - django >=2.2
+    - beautifulsoup4 >=4.8.0
+    - orjson >=3.6.0, <4.0.0
+    - shortuuid >=1.0.1, <2.0.0
+    - cachetools >=4.1.1, <5.0.0
+    - decorator >=4.4.2, <5.0.0
```

Standard formatting:
```suggestion
  host:
    - python >=3.7,<4
    - pip
    - poetry-core
  run:
    - python >=3.7,<4
    - django >=2.2
    - beautifulsoup4 >=4.8.0
    - orjson >=3.6.0,<4.0.0
    - shortuuid >=1.0.1,<2.0.0
    - cachetools >=4.1.1,<5.0.0
    - decorator >=4.4.2,<5.0.0
```